biorbd_optim/optimal_control_program.py

```diff
@@ -40,6 +40,7 @@ def __init__(
         constraints=(),
         external_forces=(),
         ode_solver=OdeSolver.RK,
+        number_of_finite_elements=5,
```

Comment:
I think that name is not verbose enough for someone not familiar with CasADi... maybe `number_integration_steps`?